src/modules/RenderHtmlLayout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import server from 'react-dom/server'
+
+/**
+ * Render the HTML layout
+ *
+ * @param head
+ * @param body & scripts
+ * @param resolverPayload
+ * @returns {string}
+ */
+export function renderHtmlLayout(head, body, resolverPayload) {
+  return '<!DOCTYPE html>' + (0, server.renderToStaticMarkup)(React.createElement(
+      'html',
+      head.htmlAttributes.toComponent(),
+      React.createElement(
+        'head',
+        null,
+        head.title.toComponent(),
+        head.meta.toComponent(),
+        head.base.toComponent(),
+        head.link.toComponent(),
+        head.script.toComponent(),
+        head.style.toComponent(),
+        React.createElement(
+          'script',
+          {
+            dangerouslySetInnerHTML: {__html: `__REACT_RESOLVER_PAYLOAD__=${resolverPayload}`}
```

Comment:
I think this needs to be `${JSON.stringify(resolverPayload)}`.

Otherwise, you get "unexpected identifier" and `[Object object]`.